File: BackendAPI/Application/DTOs/Output/OrderItemOutputDto.cs
namespace Application.DTOs.Output;

public class OrderItemOutputDto
{
    public required Guid ProductId { get; set; }
    public required string ProductName { get; set; }
    public required string ProductDescription { get; set; }
    public required string ProductImageUrl { get; set; }

    public required int Quantity { get; set; }
    public required decimal PriceAtPurchase { get; set; }
    public required DateTimeOffset OrderedAt { get; set; }
}

File: FrontendWeb/src/declarations/dtos/output/OrderItemOutputDto.ts
// OrderItemOutputDto.ts
export interface OrderItemOutputDto {
	productId: string;
	productName: string;
	productDescription: string;
	productImageUrl: string;
	companyName: string;
	quantity: number;
	priceAtPurchase: number;
	orderedAt: string;
}
